jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -744,16 +760,6 @@ private InterpreterResult executeSql(String propertyKey, String sql,
         } catch (SQLException e) { /*ignored*/ }
       }
       getJDBCConfiguration(user).removeStatement(paragraphId);
-    } catch (Throwable e) {
```

Comment:
the new finally block can throw too right?

Reply:
I thought `SQLException` is enough. 
otherwise let me fix it to `Throwable`.

Reply:
Hello, @felixcheung.
Could you check for @astroshim 's comment?

Reply:
I think there is a misunderstanding.
It was
```
  try {
        List<String> sqlArray;
...
         } finally {
               resultSet.close();
               statement.close();
 ....
         }
        }
             connection.commit();
           connection.close();
        }
        getJDBCConfiguration(user).removeStatement(paragraphId);
 -    } catch (Throwable e) {
,,,
      }
```

Now is
```
  try {
        List<String> sqlArray;
...
        } finally {
               resultSet.close();
               statement.close();
 ...
        }
 +    } catch (Throwable e) {
 +      logger.error("Cannot run " + sql, e);
 +      String errorMsg = Throwables.getStackTraceAsString(e);
 +      interpreterResult.add(errorMsg);
 +      return new InterpreterResult(Code.ERROR, interpreterResult.message());
 +    } finally {
             connection.commit();
           connection.close();
      }
```

so now there is no catch for 
```
             connection.commit();
           connection.close();

```

Reply:
@felixcheung you mean we need to return `InterpreterResult` as error when exception occurs from `connection.commit()` and `connection.close()` ?

Reply:
@felixcheung 
I think enough try-catch for `Throwable`.
This is due to the fact that the exception has been moved.
or if this is not your focus, then could you explain more to details?


Because this problem continues to cause new connections,
I hope this will be incorporated soon as possible and contribute to stabilization.

or you can handle your comments with new tickets and PR.

Thank you :)

Reply:
and there is `catch` statement for `connection.commit`.
![image](https://user-images.githubusercontent.com/3348133/30306445-2e272d8a-97b3-11e7-895b-2388f15982ef.png)